tox.ini

```diff
@@ -37,6 +37,6 @@ setenv =
     # For instance: ./.tox/py3/tmp/
     TOXTEMPDIR={envtmpdir}
 commands =
-  python -m pytest --cov-report=html --cov=west {posargs:tests} --basetemp='{envtmpdir}/pytest with space/'
+  python -m pytest -W error --cov-report=html --cov=west {posargs:tests} --basetemp='{envtmpdir}/pytest with space/'
```

Comment:
Nice, but could there be a convenient way/setting to temporarily turn this off? I mean without `git diff` and `git status` pollution.

When hacking, there is often a intermediate stage where you really want the tests to pass on work in progress that still has warnings.

Reply:
I've removed it from the `tox.ini` file and added it to the CI run.